mod/wikirate/spec/set/type/pointer/pointer_spec.rb

```diff
@@ -1,134 +1,122 @@
+    end
+    let(:elbert_punchline) do
+      create "Elbert Hubbard+punchline",
+             content: "You will never get out of it alive."
+    end
+    let(:elbert_quote) do
+      create "Elbert Hubbard+quote",
+             content: "Procrastination is the art of keeping up with yesterday."
+    end
+    let(:elbert_container) do
+      create "elbert container",
+             type_id: Card::PointerID,
+             content: "[[#{elbert.name}]]"
+    end
+
+    def json_export args={}
+      @json ||= begin
+        args[:name] ||= "normal pointer"
+        args[:type] ||= :pointer
+        if args[:content].is_a?(Array)
+          args[:content] = args[:content].to_pointer_content
+        end
+        Card::Auth.as_bot do
+          collection_card = Card.create! args
+          card = Card.create! name: "export card",
+                              type_id: Card::PointerID,
+                              content: "[[#{collection_card.name}]]"
+          card.format(:json).render_export
+        end
+      end
     end
+
     context "pointer card" do
-      it "should contain cards in the pointer card and its children" do
-        small_blind = Card.create! name: "Elbert Hubbard",
-                                   type_id: Card::BasicID,
-                                   content: "Do not take life too seriously."
-        small_blind_1 =
-          Card.create! name: "Elbert Hubbard+hello world",
-                       type_id: Card::BasicID,
-                       content: "You will never get out of it alive."
-        @collection_card =
-          Card.create! name: "normal pointer", type_id: Card::PointerID,
-                       content: "[[#{small_blind.name}]]\r\n"\
-                                "[[#{small_blind_1.name}]]"
+      it "contains cards in the pointer card and its children" do
+        json_export type: :pointer, content: [elbert.name, elbert_punchline.name]
 
-        is_expected.to include(name: "normal pointer",
-                               type: "Pointer",
-                               content: "[[Elbert Hubbard]]\n"\
-                                        "[[Elbert Hubbard+hello world]]")
-        is_expected.to include(name: "Elbert Hubbard",
-                               type: "Basic",
-                               content: "Do not take life too seriously.")
-        is_expected.to include(name: "Elbert Hubbard+hello world",
-                               type: "Basic",
-                               content: "You will never get out of it alive.")
+        expect(json_export)
+          .to include(
+                { name: "normal pointer",
+                  type: "Pointer",
+                  content: "[[Elbert Hubbard]]\n[[Elbert Hubbard+punchline]]" },
+                { name: "Elbert Hubbard",
+                  type: "Basic",
+                  content: "Do not take life too seriously." },
+                { name: "Elbert Hubbard+punchline",
```

Comment:
Maybe it's time to say good bye to our hound. Hard to find the real comments between all those woofs. Codeclimate has improved a lot and does an at least equally good maybe even better job of marking style violations.